src/neuroconv/datainterfaces/behavior/fictrac/fictracdatainterface.py

```diff
@@ -286,114 +296,88 @@ def extract_session_start_time(file_path: FilePathType) -> datetime:
 
     Parameters
     ----------
     filename : str, Path
-        Path to the configuration file.
+        Path to the configuration file in txt format.
 
     Returns
     -------
     dict
         A dictionary where the keys are the parameter names and the values are the parameter values.
 
-    Raises
-    ------
-    IOError
-        If the file cannot be read.
-    ValueError
-        If a value in the file cannot be converted to the expected type.
-
-    Examples
-    --------
-    >>> config = parse_fictrac_config('/path/to/config.txt')
-    >>> print(config['src_fn'])
-    'sample.mp4'
     """
-    import re
-
-    # Tyiping information based on https://github.com/rjdmoore/fictrac/blob/master/doc/params.md
-    type_info = {
-        "src_fn": "string OR int",
-        "vfov": "float",
-        "do_display": "bool",
-        "save_debug": "bool",
-        "save_raw": "bool",
-        "sock_host": "string",
-        "sock_port": "int",
-        "com_port": "string",
-        "com_baud": "int",
-        "fisheye": "bool",
-        "q_factor": "int",
-        "src_fps": "float",
-        "max_bad_frames": "int",
-        "opt_do_global": "bool",
-        "opt_max_err": "float",
-        "thr_ratio": "float",
-        "thr_win_pc": "float",
-        "vid_codec": "string",
-        "sphere_map_fn": "string",
-        "opt_max_evals": "int",
-        "opt_bound": "float",
-        "opt_tol": "float",
-        "c2a_cnrs_xy": "vec<int>",
-        "c2a_cnrs_yz": "vec<int>",
-        "c2a_cnrs_xz": "vec<int>",
-        "c2a_src": "string",
-        "c2a_r": "vec<float>",
-        "c2a_t": "vec<float>",
-        "roi_circ": "vec<int>",
-        "roi_c": "vec<float>",
-        "roi_r": "float",
-        "roi_ignr": "vec<vec<int>>",
-    }
 
-    # Function to parse value based on type information
-    def parse_value(value_str, type_str):
-        value_str = value_str.strip()
-        if type_str == "bool":
-            return value_str == "y"
-        elif "vec" in type_str:
-            # remove curly braces and split by comma
-            values = value_str.replace("{", "").replace("}", "").split(",")
-            if "int" in type_str:
-                return [int(val) for val in values]
-            elif "float" in type_str:
-                return [float(val) for val in values]
-        elif type_str == "int":
-            return int(value_str)
-        elif type_str == "float":
-            return float(value_str)
-        else:
-            return value_str
+    def parse_bool(x):
+        return x.lower() == "y"
+
+    def parse_vec_int(x):
+        return [int(val) for val in x.replace("{", "").replace("}", "").split(",")]
+
+    def parse_vec_float(x):
+        return [float(val) for val in x.replace("{", "").replace("}", "").split(",")]
+
+    def parse_roi_ignr(x):
+        innner_vectors = x.strip("{}").strip().split("}, {")
+        return [parse_vec_int(group) for group in innner_vectors]
+
+    KEY_PARSERS = {
```

Comment:
This isn't a global or environment variable - reason for capitalization?

Reply:
Keeping the style with the recently approved PR:
https://github.com/catalystneuro/neuroconv/blob/a4bdd8719635001cb6793d7ada154be1dac26269/src/neuroconv/datainterfaces/behavior/neuralynx/nvt_utils.py#L54-L72

But that's a good point, maybe @bendichter has some words on it.

Reply:
Guessing it's similar to what Paul uses: https://realpython.com/python-constants/#:~:text=Because%20Python%20constants%20are%20just,uppercase%20letters%20(%20A%20%E2%80%93%20Z%20)